ow_plexil/src/plans/InitializeAntennaAndArm.plp

```diff
@@ -0,0 +1,27 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Reset pan and tilt angles to 0 and stow the arm (if not already there)
+
+#include "plan-interface.h"
+
+InitializeAntennaAndArm:
+{
+  InOut Boolean initialize;
+
+  log_info ("Initializing lander antenna and arm...");
+
+  log_info ("Resetting tilt angle to 0...");
+  SynchronousCommand tilt_antenna (0);
+  log_info ("Resetting pan angle to 0...");
+  SynchronousCommand pan_antenna (0);
+  log_info ("Stowing the arm...");
+  SynchronousCommand stow();
+
+  log_info ("Initialization complete.");
+  initialize = false;
+
+  Wait 5; // To buffer before a plan begins after initialization
```

Comment:
Why is this needed?

Reply:
After running the plans a bunch sometimes I felt it wasn't visually clear when antenna actions were due to initialization vs. due to the start of the plan itself. Adding a wait gives the user time to process the printed output and visually see that the lander is in the zero positions before it begins moving to execute the plan. I know what these plans will do, but if someone is using this initialization node and writing a new plan it might be easier to debug having a separation like this. That was my logic, but I would be happy to remove it if you think it is not helpful.

Reply:
The plan does print "Initialization complete", which should delineate this with the test part.  You could reword it to something like "Arm and antenna initialized.".   Waits are best avoided unless they are programatically needed, e.g. giving ROS time to respond or publish a message.

Reply:
Makes sense, I removed the wait.